docs/docs/guides/tutorials/Nextjs.mdx

```diff
@@ -0,0 +1,750 @@
+        <hanko-auth />
+      </div>
+    </div>
+  );
+}
+
+```
+
+## Time to display the right Todos 🪄✨
+Lastly, we should only display the todos for the user that is logged in. To do so, we need to link the todos to the correct "user ID". The first step is to update the Todo model in the `prisma schema`:
+
+```javascript
+model Todo {
+  userId String
+  id String @id @default(uuid())
+  title String
+  complete Boolean
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
+```
+Then run the following command to create a migration:
+
+```
+npx prisma migrate
+```
+Or the following to push the schema changes directly to the database:
+
+```
+npx prisma db push
+```
+Next step is to update the `api/todo/route.tsx` file to get the user ID from the token, then create a new todo if there is a user ID:
+
+```javascript
+import { NextResponse } from "next/server";
+import { cookies } from "next/headers";
+import * as jose from "jose";
+import { prisma } from "@/db";
+
+export async function userId() {
+  const token = cookies().get("hanko")?.value;
+  const payload = jose.decodeJwt(token ?? "");
+
+  return payload.sub;
+}
+
+export async function POST(req: Request) {
+  const userID = await userId();
+  const { title } = await req.json();
+
+  if (userID) {
+    if (typeof title !== "string" || title.length === 0) {
+      throw new Error("That can't be a title");
+    }
+    await prisma.todo.create({
+      data: { title, complete: false, userId: userID ?? "" },
+    });
+
+    return NextResponse.json({ message: "Created Todo" }, { status: 200 });
+  } else {
+    return NextResponse.json({ error: "Not Found" }, { status: 404 });
+  }
+}
+
+```
+
+The final step is to update the Prisma call to fetch all the todos from the `todo/page.tsx`:
+
+```javascript
+import { Logout } from "@/components/Logout";
+import { Profile } from "@/components/Profile";
+import { NewTodo } from "@/components/todos/NewTodo";
+import { TodoItem } from "@/components/todos/TodoItem";
+import { prisma } from "@/db";
+import { userId } from "../api/todo/route";
+
+export default async function Todo() {
+  const userID = await userId();
+
+  const todos = await prisma.todo.findMany({
+    where: {
+      userId: { equals: userID },
+    },
+  });
+
+  return (
+    <main className=" flex flex-col min-h-screen justify-center items-center bg-slate-50 relative ">
+      <div className="absolute top-4 left-16">
+        <div className=" relative py-4 space-x-6">
+          <Profile />
+          <Logout />
+        </div>
+      </div>
+      <div className="bg-slate-300 rounded-3xl py-6  h-[400px] w-[450px] flex flex-col text-slate-800">
+        <h1 className="text-3xl text-center">My to dos</h1>
+        <NewTodo />
+        <ul className="px-6">
+          <TodoItem todos={todos} />
+        </ul>
+      </div>
+    </main>
+  );
+}
+
+```
+
+That’s all, you’ve successfully created a Todo app with Next.js, Hanko, Prisma and Tailwind CSS! 
```

Comment:
Also, be sure to run it locally and test if everythings looking good. You can go to docs directory, run `npm install` and `npm run start`